### ISSUE TITLE
CRASH !! 

### ISSUE THREAD

**Issue by PrashantKT** [saeid/GDGauge#4]
Hey I got crash  with sample project. 

Crash in this function 

```
public func resetColors(){
        UIView.animate(withDuration: 0.4) {
            self.baseCircleShape.strokeColor = self.baseColor.cgColor // Crash here  because self.baseCircleShape is nil 
            self.handleShape.fillColor = self.handleColor.cgColor
            for layer in self.firstIndicatorsSerie{
                layer.strokeColor = self.sepratorColor.cgColor
            }
            for layer in self.secondIndicatorsSerie{
                layer.strokeColor = self.sepratorColor.cgColor
            }
        }
    }

```
How to reproduce 
---

In sample project 

uncomment this line ` speed.showBorder = false`

**Comment by saeid** [comment]
Thanks for reporting the issue. I'm gonna take a look into this!

**Comment by saeid** [comment]
@PrashantKT This should be fixed in `1.1.4`. 
run `Pod update` to update pods as well.
